content/en/docs/Contribute/_index.md

```diff
@@ -0,0 +1,528 @@
+1. Make the changes requested.
+1. Commit the changes.
+
+When your review is complete, a reviewer merges your PR and your changes go liv
+a few minutes later.
+
+### Work from a local fork {#fork-the-repo}
+
+If you're more experienced with Git, or if your changes are larger than a few
+lines, work from a local fork.
+
+Make sure you have
+[git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) installed
+on your computer. You can also use a user interface for Git.
+
+Figure 2 shows the steps to follow when you work from a local fork. The details
+for each step follow.
+
+```mermaid
+flowchart LR
+1[Fork the open-telemetry/opentelemetry<br>repository] --> 2[Create local clone<br>and set upstream]
+subgraph changes[Your changes]
+direction TB
+S[ ] -.-
+3[Create a branch<br>example: my_new_branch] --> 3a[Make changes using<br>a text editor] --> 4["Preview your changes<br>locally using Hugo<br>(localhost:1313)"]
+end
+subgraph changes2[Commit / Push]
+direction TB
+T[ ] -.-
+5[Commit your changes] --> 6[Push commit to<br>origin/my_new_branch]
+end
+
+2 --> changes --> changes2
+
+classDef grey fill:#dddddd,stroke:#ffffff,stroke-width:px,color:#000000, font-size:15px;
+classDef white fill:#ffffff,stroke:#000,stroke-width:px,color:#000,font-weight:bold
+classDef k8s fill:#326ce5,stroke:#fff,stroke-width:1px,color:#fff;
+classDef spacewhite fill:#ffffff,stroke:#fff,stroke-width:0px,color:#000
+class 1,2,3,3a,4,5,6 grey
+class S,T spacewhite
+class changes,changes2 white
+```
+
+Figure 2. Working from a local fork to make your changes.
+
+#### Fork the opentelemetry.io repository
+
+1. Navigate to the
+   [`opentelemetry.io`](https://github.com/open-telemetry/opentelemetry.io/)
+   repository.
+1. Select **Fork**.
+
+#### Create a local clone and set the upstream
+
+1. In a terminal window, clone your fork and install the requirements:
+
+   ```shell
+   git clone git@github.com:<your_github_username>/opentelemetry.io.git
+   cd opentelemetry.io
+   npm install
+   ```
+
+1. Set the `open-telemetry/opentelemetry.io` repository as the `upstream`
+   remote:
+
+   ```shell
+   git remote add upstream https://github.com/open-telemetry/opentelemetry.io.git
+   ```
+
+1. Confirm your `origin` and `upstream` repositories:
+
+   ```shell
+   git remote -v
+   ```
+
+   Output is similar to:
+
+   ```none
+   origin	git@github.com:<your_github_username>/opentelemetry.io.git (fetch)
+   origin	git@github.com:<your_github_username>/opentelemetry.io.git (push)
+   upstream	https://github.com/open-telemetry/opentelemetry.io.git (fetch)
+   upstream	https://github.com/open-telemetry/opentelemetry.io.git (push)
+   ```
+
+1. Fetch commits from your fork's `origin/main` and
+   `open-telemetry/opentelemetry.io`'s `upstream/main`:
+
+   ```shell
+   git fetch origin
+   git fetch upstream
+   ```
+
+   This makes sure your local repository is up to date before you start making
+   changes.
```

Comment:
You also need to push changes to upstream to origin. 

Personally, I stopped working that way, I _rarely_ add an upstream, I keep my fork up-to-date (either manually or with an github app, etc.) and only pull from the fork. PRs are still send to upstream.

Reply:
Hmm, how do you keep the fork in sync with upstream if git doesn't know where to pull?